foodsaving/groups/templates/user_inactive_in_group.subject.jinja2

```diff
@@ -0,0 +1 @@
+[{{ site_name }}] {% trans %}We're missing you!{% endtrans %}
```

Comment:
I guess the subject should include the group name, something like "{{ group_name }} is missing you!"